### PR TITLE
Triage 2024 01 23

### PR DESCRIPTION
**Pull request by Kobzol**
Perf. triage for 2024-01-23.

### CODE REVIEW

File: triage/2024-01-23.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.7%  | [0.7%, 0.7%]   | 1     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -1.2% | [-8.2%, -0.2%] | 161   |
+| Improvements ✅ <br /> (secondary) | -1.5% | [-3.7%, -0.3%] | 64    |
+| All ❌✅ (primary)                 | -1.2% | [-8.2%, 0.7%]  | 162   |
+
+* Incredible wins across many benchmarks, caused by improving hashing in the compiler.
```
Comment (nnethercote) [inline triage/2024-01-23.md:97]: Is it worth mentioning that icount wins were fantastic, but the wall-time wins were 3x better? That's a rare combination.

**Review by nnethercote — APPROVED** [review]
Thanks!

File: triage/2024-01-23.md (hunk 1 above)
Comment (Kobzol) [reply triage/2024-01-23.md:97]: Good point!

Comment (Kobzol) [reply triage/2024-01-23.md:97]: Although, I'd also have to modify the TWiR PR, since it links to the old commit :sweat_smile:

**Pull request merged**